packages/vaadin-upload/src/vaadin-upload.js

```diff
@@ -408,9 +408,16 @@ class UploadElement extends ElementMixin(ThemableMixin(PolymerElement)) {
                 unknown: 'unknown remaining time'
               },
               error: {
-                serverUnavailable: 'Server Unavailable',
-                unexpectedServerError: 'Unexpected Server Error',
-                forbidden: 'Forbidden'
+                serverError: {
+                  internalServerError: "File couldn't be uploaded becase there is an issue with server.",
+                  serviceNotAvailable: "File couldn't be uploaded becase service is not availble."
```

Comment:
`File cannot be uploaded because the service is not available.`

`File cannot be uploaded because of the internal server error.`

`You are not authorized to upload the file.`

`Server cannot process your request`.